projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -213,6 +214,13 @@ function render_newsletter_access_rows( $column_id, $post_id ) {
 	}
 }
 
+/**
+ * Adds the Newsletter column styles
+ */
+function newsletter_access_column_styles() {
+	echo '<style id="jetpack-newsletter-newsletter-access-column"> table.fixed .column-newsletter_access { width: 10%; } </style>';
```

Comment:
Noticed that almost all inline styles have an id. Not sure if it's needed for admin but decided to add it.